core/src/test/scala/org/apache/spark/deploy/history/EventLogFileCompactorSuite.scala

```diff
@@ -283,6 +284,24 @@ class EventLogFileCompactorSuite extends SparkFunSuite {
     }
   }
 
+  test("Use the value of spark.eventLog.compression.codec set by user") {
```

Comment:
Check this test without change of `EventLogFileWriters.scala`  and `EventLogFileCompactor.scala`, the new test cases will also pass. Can you check the design of the new test? If this is a bug fix, the new test should fail without change of `EventLogFileWriters.scala`  and `EventLogFileCompactor.scala`

Reply:
In addition, please use a test prefix like the following.
```scala
- test("Use the value of spark.eventLog.compression.codec set by user") {
+ test("SPARK-43991: Use the value of spark.eventLog.compression.codec set by user") {
```

Reply:
> @shuyouZZ I checked this test without change of `EventLogFileWriters.scala` and `EventLogFileCompactor.scala`, the new test cases will also pass. Can you check the design of the new test? If this is a bug fix, the new test should fail without change of `EventLogFileWriters.scala` and `EventLogFileCompactor.scala`

oh, it's my mistake. The unit test has been fixed, thanks for your review!

Reply:
> In addition, please use a test prefix like the following.
> 
> ```scala
> - test("Use the value of spark.eventLog.compression.codec set by user") {
> + test("SPARK-43991: Use the value of spark.eventLog.compression.codec set by user") {
> ```

Noted, thanks for your advice!